Barkane/ProjectSettings/Packages/com.unity.probuilder/Settings.json

```diff
@@ -37,6 +37,11 @@
                 "type": "System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
                 "key": "experimental.enabled",
                 "value": "{\"m_Value\":false}"
+            },
+            {
```

Comment:
remove these changes. sorry this should be in gitignore, but we are making the gitignore change on main branch, so just removing the changes should be good enough